airflow/providers/openlineage/extractors/base.py

```diff
@@ -84,6 +84,12 @@ def get_operator_classnames(cls) -> list[str]:
     def extract(self) -> OperatorLineage | None:
         try:
             return self._get_openlineage_facets(self.operator.get_openlineage_facets_on_start)  # type: ignore
+        except ImportError:
+            self.log.error(
+                "OpenLineage provider method failed to import OpenLineage integration. "
```

Comment:
Should we not ask the user to install open lineage here?

Reply:
So this is somehow a failsafe that might not make sense - but I think it's better than having try-except chains in the implementation method itself. 

Basically we call OL method that imports OL modules, and call it from OL integration - so it should never raise exception, in theory.